src/codegen/codegen_cuda.cc

```diff
@@ -83,7 +87,8 @@ void CodeGenCUDA::PrintType(Type t, std::ostream& os) {  // NOLINT(*)
     }
     if (t.bits() == 8 && t.lanes() == 4) {
       // directly 4 8 bit int in integer.
-      os << "int"; return;
+      enable_int8_ = true;
+      os << "char4"; return;
```

Comment:
@tqchen do we need to support other lanes size? e.g. int4 if lanes == 16

Reply:
This, this would actually be very helpful, @vinx13 can you elaborate what are possible things we need to support and value translation rules in here?